### ISSUE TITLE
Feature Request: Respect Atom's tab and tab-width settings when prettifying

### ISSUE THREAD

**Issue by schnitzelK1ng** [federomero/pretty-json#72]
It would be nice if it used hard/soft tabs and use tab-width specified in Atom's settings.

**Comment by stereonom** [comment]
I modified the `formatter.js` to get this working.

```
  space (scope) {
    const softTabs = atom.config.get('editor.softTabs', {scope})
    const tabLength = Number([atom.config.get('editor.tabLength', {scope})])
    if (softTabs) {
      return Array(tabLength + 1).join(' ')
    } else {
      return '\t'
    }
  },
```

**Comment by lexicalunit** [comment]
Thanks for the fix! Published as  `pretty-json@v2.0.4`.

**Comment by stereonom** [comment]
Hey.
You forgot to change the `if (softTabs != null) {` to ` if (softTabs) {` 
The Editor-Config `editor.softTabs` always gives u a boolean.

So, it's still broken in the current version.

**Comment by lexicalunit** [comment]
Fixed and released as `pretty-json@v2.1.2` 🎉
